lighthouse-core/report/html/report-styles.css

```diff
@@ -100,6 +100,8 @@
   --color-white: #FFFFFF;
   --color-blue-200: #90CAF9;
   --color-blue-900: #0D47A1;
+  --color-teal-300: #00d6c1;
+  --color-teal-700: #008577;
```

Comment:
Re-introduced teal since it was a bit nicer than the green. Do the numbers at the end have a specific meaning, other than broadly identifying darkness?

Reply:
It's the shade in the material design color palette https://material.io/design/color/#

Reply:
is this the right number, then?

Reply:
Changed it to use material design colors.

Also cut the snippet line height a bit.

<img width="895" alt="Screenshot 2019-05-20 at 20 31 26" src="https://user-images.githubusercontent.com/1303660/58046890-83be8200-7b3e-11e9-8b69-3bcd63068ab1.png">
<img width="879" alt="Screenshot 2019-05-20 at 20 31 33" src="https://user-images.githubusercontent.com/1303660/58046891-83be8200-7b3e-11e9-8803-3ce53e6de3df.png">